web/template/dashboard.gotmpl

```diff
@@ -10,23 +10,35 @@
       Contents
     </h2>
     <ul class="govuk-tabs__list">
-      <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+      <li class="govuk-tabs__list-item{{ if eq .Path "/pending-cases" }} govuk-tabs__list-item--selected{{ end }}">
```

Comment:
It will probably be easier to have 2 templates, especially as one will end up with loads of filters on it